api/services/PassportService.js

```diff
@@ -144,6 +144,39 @@ module.exports = class PassportService extends Service {
   }
 
   /**
+   * Update the local passport password of an user
+   * @param user
+   * @param password
+   * @returns {Promise}
+   */
+  updateLocalPassword(user, password) {
+    const criteria = (user._id) ? {_id: user.id} : {id: user.id} //eslint-disable-line
+
+    return this.app.services.FootprintService.find('user', criteria, {populate: 'passports'})
+      .then(user => {
+        if (user && user.length > 0) {
+          user = user[0]
+          if (user.passports) {
+            const localPassport = user.passports.find(passportObj => passportObj.protocol === 'local')
+            if (localPassport) {
+              localPassport.password = password
+              return localPassport.save()
+            }
+            else {
+              return Promise.resolve()
+            }
+          }
+          else {
+            return Promise.reject('no_available_passports')
```

Comment:
for errors we should at least create an error object `new Error('no_available_passports')`, I have to change errors to be more consistent and more compatible with Trails error handling but if you can use this as example https://github.com/jaumard/trailpack-passport/blob/master/api/services/PassportService.js#L190.
Thanks :)